ext/UnrolledUtilitiesStaticArraysExt.jl

```diff
@@ -0,0 +1,10 @@
+module UnrolledUtilitiesStaticArraysExt
+
+import UnrolledUtilities
+import StaticArrays: SVector
+
+UnrolledUtilities.length_from_type(::Type{<:SVector{N}}) where {N} = N
+UnrolledUtilities.target_output_type(::SVector) = SVector
+UnrolledUtilities.output_constructor(::Type{SVector}) = SVector
```

Comment:
```suggestion
UnrolledUtilities.output_constructor(::Type{<:SVector}) = SVector
```
Should this subtyping?

Reply:
The input to `output_constructor` will always be a type returned by `target_output_type`. The type in this case is just `SVector`, so we don't need to subtype here. Same for `Tuple` and `LazySequence`, though we need to subtype for `BitSequence` to propagate the unsigned word type.